apps/src/templates/courseOverview/ResourcesEditor.js

```diff
@@ -73,23 +97,21 @@ export default class ResourcesEditor extends Component {
         )}
 
         <div style={styles.box}>
+          <div style={styles.error}>{errorString}</div>
           <div style={{marginBottom: 5}}>Preview:</div>
-          <CourseOverviewTopRow
-            sectionsInfo={[]}
-            id={-1}
-            title="Unused title"
-            resources={resources.filter(x => !!x.type)}
-          />
+          {this.props.renderPreview(resources.filter(x => !!x.type))}
```

Comment:
I had initially passed in a `isCourse` prop, and then rendered had a conditional here that either rendered a CourseOverviewTopRow or a DropdownButton, but then decided it was better to let the caller own preview rendering. Ultimately I'm not sure either approach is clearly better than the other.